packages/core/src/Stores/traders-hub-store.js

```diff
@@ -395,6 +397,12 @@ export default class TradersHubStore extends BaseStore {
         this.setCombinedCFDMT5Accounts();
     }
 
+    get restricted_countries_filter_content() {
```

Comment:
@aizad-deriv 

![image](https://user-images.githubusercontent.com/72082844/233277736-fc1a417b-7e77-4c70-ab12-ec3c8f0a25ef.png)

Reply:
i put in next commit 😂

Reply:
🐵 👍🏼